### PR TITLE
#249 Migrate static docs to new GH pages site

### PR DESCRIPTION
**Pull request by TheCoolBlackCat**
## Description

* Migrated the static documentation branch to a `docs/` folder
* Added CI to publish the static docs to GH pages

(#249) Starts the process of migrating to docs generated automatically, by creating a static site hosted on GH pages

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] My commits follow the [commitizen](https://github.com/commitizen/cz-cli#philosophy) commit convention
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by TheCoolBlackCat** [comment]
@jwu910 I'm not sure if this will need an admin to turn on GH pages in the config, or if the CI tweak is enough but this should deploy the static docs. I'd like to add static generation from the README in CI at a later date if this PR LGTY?

**Comment by jwu910** [comment]
Oh sweet! Thanks for opening this PR! I'll try to take a look when I get home!

File: docs/CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+checkit.club
```
Comment (jwu910) [inline docs/CNAME:1]: Oh I know for sure we can drop this though. I stopped paying for the domain, so no custom domain anymore

**Review by jwu910 — COMMENTED** [review]
Just one minor request I can see from my phone, the CNAME might cause issues while I'm testing since I don't own checkit.club anymore.

I'll take another look when I get more time. Thanks again!

File: docs/CNAME (hunk 1 above)
Comment (TheCoolBlackCat) [reply docs/CNAME]: I just removed this file, as its not needed :)

**Review by jwu910 — APPROVED** [review]
You know i have no clue how im going to go about testing this, i think i will just merge this in and figure it out.... Not touching any of the js code, should be fine. 🖖🏼

**Comment by jwu910** [comment]
@all-contributors please add @TheCoolBlackCat for code, docs

**Pull request merged**